packages/sprites/README.md

```diff
@@ -52,4 +50,9 @@ basemaps-sprites --extension .png ./config/sprites/topographic
 
 # Load png, webp and svg sprites
 basemaps-sprites --extension .png --extension .svg --extension .webp ./config/sprites/topographic
-```
\ No newline at end of file
+```
+
+# Refference of the test sprites in the unit tests.
+
+Some test sprites from openstreetmap
```

Comment:
This needs to be way more specific 

Can we say exactly which files and link to their github and not the PNG?

https://github.com/ZeLonewolf/openstreetmap-americana/

Reply:
Updated, thanks.